modules/api/README.md

```diff
@@ -0,0 +1,50 @@
+# API
+
+## Purpose
+
+The api module is intended to provide a stable, versioned REST interface
```

Comment:
I don't understand.. it's already written api?

Reply:
possibly that in line 1, `API` is used while in line 5, it is `api`. Have a constant way to write it across the files (every files in LORIS).
As it is an abbreviation, it should be `API`.

Reply:
hahaha, I meant either mark `api` in code format or if not in code format, write it in capital letter however, not a blocker.

Reply:
I had written it lowercase because I thought all the README's started "The xxx module" where xxx is the lower case directory name, but I just checked a couple randomly and it seems to be very inconsistent, so I'll capitalize it since it looks better.